tests/FlightAsyncTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace tests;
+
+use Flight;
+use flight\Engine;
+use PHPUnit\Framework\TestCase;
+
+class FlightAsyncTest extends TestCase
+{
+    public static function setUpBeforeClass(): void
+    {
+        Flight::setEngine(new Engine());
+    }
+
+    protected function setUp(): void
+    {
+        $_SERVER = [];
+        $_REQUEST = [];
+    }
+
+    protected function tearDown(): void
+    {
+        unset($_REQUEST);
```

Comment:
Here need to be the same that `setUp()`

`$_REQUEST` and `$_SERVER` , always exist, by default is an empty array [].

```
$_SERVER = [];
$_REQUEST = [];
```